stixcore/products/product.py

```diff
@@ -97,12 +97,17 @@ def __call__(self, *args, **kwargs):
                 control = QTable.read(file_path, hdu='CONTROL')
                 data = QTable.read(file_path, hdu='DATA')
 
+                scet_timerange = SCETimeRange(start=SCETime.from_string(header.get('OBT_BEG')),
+                                              end=SCETime.from_string(header.get('OBT_END')))
+
                 if level != 'LB':
                     if timesys == 'OBT':
-                        obs_beg = SCETime.from_string(header['OBT_BEG'])
-                        offset = obs_beg.as_float()
+                        try:
+                            offset = SCETime.from_string(header['OBT_BEG'], sep=':')
+                        except ValueError:
+                            offset = SCETime.from_string(header['OBT_BEG'], sep='f')
```

Comment:
the "f" as delimiter was totally removed again in pr #109 as it was not used in filenames any more. the SCETime in filenames should be only coarse